packages/webcert/src/feature/certificate/UnifiedEdit/UeInteger/UeInteger.tsx

```diff
@@ -0,0 +1,82 @@
+import {
+  CertificateDataElement,
+  ConfigUeInteger,
+  QuestionValidationTexts,
+  TextInput,
+  ValidationError,
+  ValueInteger,
+} from '@frontend/common'
+import { ValidationWrapper } from '@frontend/common/src/components/Inputs/DatePickerCustom/Styles'
+import _ from 'lodash'
+import * as React from 'react'
+import { useState } from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+import { updateCertificateDataElement, updateClientValidationError } from '../../../../store/certificate/certificateActions'
+import { getVisibleValidationErrors } from '../../../../store/certificate/certificateSelectors'
+
+const INVALID_PERCENT_PERIOD_ERROR = 'Ange ett värde mellan 0 och 100 %'
+
+export interface Props {
+  question: CertificateDataElement
+  disabled: boolean
+}
+
+const UeInteger: React.FC<Props> = ({ question, disabled }) => {
+  const dispatch = useDispatch()
+  const questionValue = question.value as ValueInteger
+  const questionConfig = question.config as ConfigUeInteger
+  const [percent, setPercent] = useState<string>('')
```

Comment:
This component should handle all kinds of integers, not just percent. A more general name is preferable